### DISCUSSION TITLE
DAI 2024-2025 - Practical work 3 - Thirusan Rajadurai, Raphaël Perret

### DISCUSSION THREAD

**Discussion by Thiru-sama** [heig-vd-dai-course/heig-vd-dai-course#532] [Show and tell]
We will create an application to manage a food truck specialized in pizzas.
It will be used to simulate ( but can be used in a real case ) a pizza truck's orders.
The main idea is that the server start will represent a shift's start.
Customers can :
- See the menu with available pizzas
- Order a Pizza
- (Change the pizza type before it is ready)
- See if the ordered pizza is ready
- Retrieve the pizza when ready

The Vendor can :
- See the orders to prepare
- (Add a pizza to the menu)
- (Remove a pizza from the menu)

**Comment by GeraudSilvestri** [comment]
Good for me

**Comment by rp2709** [comment]
Repo : https://github.com/rp2709/DAI-Practical-Work-3
Hash : aa12b76bd4b4baacf621327d18af8feee57ee9fa
